README.md

```diff
@@ -1,78 +1,157 @@
-# awesome-books-ES6-modules
```

Comment:
- [ ] I am requesting this change because you did not change the affected file.

  In your `index.js` file, kindly refactor the `showBlock` method to an arrow function in keeping with the ES6 syntax

  ![dddidi](https://user-images.githubusercontent.com/36542986/205605770-ecc7df57-2af4-462d-8a90-2f74de2e43aa.jpg)

Reply:
well resolved! ✔️